src/jit/inline.h

```diff
@@ -13,6 +13,7 @@
 // InlineTarget -- enum, target of a particular observation
 // InlineImpact -- enum, impact of a particular observation
 // InlineObservation -- enum, facts observed when considering an inline
+// InlineContext -- class, remembers what inlines happened
```

Comment:
Sounds like a good idea. I have more edits coming here so will add this onto the next bit of work.